beps/0001-notifications-system/README.md

```diff
@@ -165,11 +167,12 @@ The components that are presented with dashed lines are not directly part of thi
 
 Let's walk through the process of sending of a notification to a user:
 
-1. A backend plugin that wants to send a notification to a user uses the `NotificationService`, which in turn makes a POST request to the notification backend. The request contains a filter to select the target users, as well as the notification payload.
+1. A backend plugin that wants to send a notification to a user uses the `NotificationService`, which in turn makes a POST request to the notification backend. The request contains either a filter to select the target users or type `broadcast`, as well as the notification payload.
 
    1b. As the notification backend handles the request it could optionally invoke a series of notification processors. These processors can be used to transform the notification payload, send the notification through other channels, and decide whether the regular notification flow should continue or not. This step is not in scope for this proposal.
 
 2. The notification backend stores the received notification in its database. At this point the notification is available from the notification backend's read API.
+
```

Comment:
It seems reasonable to request signal events about broadcasted messages.

@drodil @Rugvip What about extending the Signal API to support broadcasts?
Generating events per logged-in user seems odd.

Reply:
Example of broadcast in this PR: https://github.com/backstage/backstage/pull/23194